cmd/jiralert/main.go

```diff
@@ -94,16 +97,27 @@ func main() {
 		level.Debug(logger).Log("msg", "  matched receiver", "receiver", conf.Name)
 
 		// TODO: Consider reusing notifiers or just jira clients to reuse connections.
-		tp := jira.BasicAuthTransport{
-			Username: conf.User,
-			Password: string(conf.Password),
+
+		// if cert and key are specified, ignore username/password
```

Comment:
Can we have full sentence in comments?

(Capital letter and trailing period)

Reply:
Will do, though that's an incorrect comment at this point - will remove